employee-rostering-backend/src/main/resources/application.properties

```diff
@@ -1,2 +1,8 @@
+# Setting datasource
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.url=jdbc:h2:mem:testdb
```

Comment:
Why do we need this? Is it for the Benchmark>

Reply:
This is not needed, Spring Boot autoconfigures the DataSource. I added it to make sure h2 is used for testing purposes, but I can remove it.